pytest-embedded-serial-esp/pytest_embedded_serial_esp/serial.py

```diff
@@ -27,6 +27,7 @@ def __init__(
         target: Optional[str] = None,
         beta_target: Optional[str] = None,
         port: Optional[str] = None,
+        port_location: Optional[str] = None,
```

Comment:
no need here.

Reply:
The kwargs that is not been captured in this subclass `EspSerial` would pass to `Serial` here: https://github.com/espressif/pytest-embedded/blob/main/pytest-embedded-serial-esp/pytest_embedded_serial_esp/serial.py#L117